src/styles/Components/Introduction.sass

```diff
@@ -0,0 +1,160 @@
+.introduction
+  width: 100%
+  min-height: 100vh
+  background-image: url('../../../src/assets/images/beach.jpg')
+  background-size: cover
+  background-repeat: no-repeat
+  opacity: 0.9
+  overflow-x: hidden
+  color: white
+
+  &__shadow
+    position: absolute
+    left: 0px
+    right: 0px
+    width: 100%
+    height: 100px
+    background: rgb(195,193,230)
+    background: linear-gradient(176deg, rgba(195,193,230,1) 0%, rgba(238,238,244,0.19371498599439774) 0%, rgba(255,255,255,1) 100%)
+
+  &__wrapper
+    width: 100%
+    padding: 100px 150px 
+
+  &__slider
+    display: flex
+    justify-content: center
+    align-items: center
+    flex-direction: column
+    width: 100%
+
+    &--description
+      font-size: 20px
+      align-self: flex-start
+      padding-bottom: 20px
+      letter-spacing: 1.5px
+      border-bottom: 3px solid white
+
+
+    &__tab-wrapper
+      width: 100%
+      align-self: flex-end
+      display: flex
+
+      span
+        display: block
+        width: 100%
+        height: 2px
+        border: .5px solid rgba(256,256,256, 0.3)
```

Comment:
there is no such color, the range goes `0 - 255`. You can omit the `0` when passing alpha, so `.3` is fine